kore/test/Test/Kore/Simplify/Predicate.hs

```diff
@@ -162,6 +166,43 @@ test_simplify =
             (fromCeil_ (mkEquals Mock.testSort fa fb))
             [[fromEquals_ fa fb]]
         ]
+    , testGroup
+        "\\floor"
+        [ test
+            "\\bottom"
+            (fromFloor_ (mkBottom Mock.testSort))
```

Comment:
Is `fromFloor_` an abstraction over
a conversion followed by an attribute synthesizing?

Reply:
Yes, something like that, it uses the common `Syntax` types to build terms in a more general way. It's defined here: https://github.com/kframework/kore/blob/55f677c31a2617df35227b3c0752f1cfa77b3fd5/kore/src/Kore/Internal/From.hs#L135